src/Microsoft.DotNet.Darc/src/Darc/Helpers/UxHelpers.cs

```diff
@@ -269,7 +269,8 @@ public static async Task<bool> VerifyAndConfirmBranchExistsAsync(IRemote remote,
             {
                 branch = GitHelpers.NormalizeBranchName(branch);
 
-                await remote.GetDependenciesAsync(repo, branch);
+                var dependencies = await remote.GetDependenciesAsync(repo, branch);
```

Comment:
There shouldn't be any changes in this file.

Reply:
same here.